src/components/Argument/components/EditableArgument.js

```diff
@@ -52,15 +60,16 @@ function EditableArgument(props) {
   return (
     <div className={'h5p-category-task-editable-container'}>
       <button
+        ref={buttonRef}
         className={classnames('h5p-category-task-editable-button', {
-          'h5p-category-task-editable-button--editing': inEditMode === true,
+          'hidden': inEditMode === true,
         })}
         onClick={startEditing}
       >
-        <span className={'visible-hidden'}>{'Edit argument ' + props.argument}</span>
+        <span className={'visible-hidden'}>{`${translate('editArgument')} ${argument}`}</span>
```

Comment:
I wonder if we should include the argument within the translation, as the order is different in some languages.

![image](https://github.com/NDLANO/h5p-category-task/assets/9085189/1368c38d-f6c0-4364-b619-707b8f978633)

Reply:
There's also an SR only `Argument` on line 78 that we should translate.

Reply:
Even though the order of "edit" and "argument" is different, I don't think we need to include the actual argument text within the translation for now. But we can later if we see the need to.